src/Migrations/M020MigrateCommentsSettings.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace P4\MasterTheme\Migrations;
+
+use P4\MasterTheme\Settings\CommentsGdpr;
+
+class M020MigrateCommentsSettings
+{
+    /**
+     * Perform the actual migration.
+     *
+     * @param MigrationRecord $record Information on the execution, can be used to add logs.
+     * phpcs:disable SlevomatCodingStandard.Functions.UnusedParameter -- interface implementation
+     */
+    public static function execute(): void
+    {
+        $value = planet4_get_option('gdpr_checkbox');
```

Comment:
Hi @mleray , While testing on local I found that, the old `gdpr_checkbox` value is stored in `planet4_comments` option (`get_option('planet4_comments')`)  Could you please double check once?

Reply:
This is what I get on `main` branch for `get_option('planet4_comments')`: 
```
array(1) { ["gdpr_checkbox"]=> string(2) "on" }
```
and this is what I get for `get_option('gdpr_checkbox')`:
```
string(2) "on"
```

But actually `planet4_get_option` returns null with both of these keys 🤔 So I guess here I should use `get_option('gdpr_checkbox')` right?

Reply:
When I toggle the "Comments opt-in checkbox" setting(on `main` branch) the `planet4_comments` (i.e `get_option('planet4_comments')`) got change. so we need to consider `get_option('gdpr_checkbox')` value.

Reply:
I've updated the PR, let me know what you think!